minionsai/discriminator_only/translator.py

```diff
@@ -1,11 +1,21 @@
 
 
 from functools import lru_cache
+from operator import truediv
+from tkinter.tix import MAX
```

Comment:
What is tkinter.tix? Isn't that some plotting library? I suspect your editor added this without your permission and it's unused. Also the xmlrpc thing below.